projects/uswds-components/src/lib/form-field/form-field.component.html

```diff
@@ -0,0 +1,15 @@
+<div #labelDiv [class.usa-form-group--error]="!!errorMessage" class="usa-form-group ">
+  <label *ngIf="!!label" [attr.for]="name" class="usa-label" [class.usa-input-error-label]="errorMessage">
+    {{ label || '' }}
+    <span *ngIf="required || requiredFlag">
+      (Required)
+    </span>
+    <ng-content select="[label-right]"></ng-content>
```

Comment:
Is this label-right meant for potential icons to the right of label? I don't see a selector created for this in the changes

Reply:
Its not used removed it